### PR TITLE
#94: interaction documentation for onBlockDestroyedByPlayer

### PR DESCRIPTION
**Pull request by G-Rath**
#94 : Added documentation about the `onBlockDestroyedByPlayer` interaction method.

### CODE REVIEW

File: docs/blocks/interaction.md [hunk 1]
```diff
@@ -74,4 +74,34 @@ Entity Collision
...
+    The `pos` parameter may not hold the state indicated
+
+### Usage example
+This method is perfect for adding custom events as a result of a player destroying a block
+
+By default this method does nothing.  
+
+The **TNT Block** overrides this method to cause it's explosion when a player destroys it.  
+This method is used by extended pistons; since an extended piston is made up of two blocks (the extended head and the base), 
+this method is ued by the **PistonMoving Block** to destroy the base block when it's destroyed.
```
Comment (MinecraftPlaye) [inline docs/blocks/interaction.md]: `ued` --> `used`

Comment (G-Rath) [inline docs/blocks/interaction.md]: Great catch, thank! I've reworded that sentence slightly, as I didn't like the `when it's destroyed` - Hopefully the new phrasing is a bit clearer (at the cost of being slightly repetitive, but I think it's worth it for the sake of clarity).

**Comment by tterrag1098** [comment]
Would be better to have small changes like this as separate commits in a single PR...because now your others are stuck on merge conflicts.

**Comment by G-Rath** [comment]
@MinecraftPlaye ehhh yes that would be have been better. 

I did see the conflict coming, but I wasn't sure how many I'd do, since I'm new to the whole MCForge game :P

**Comment by Akxe** [comment]
Add sideness information about this event, is it server event, client, or both?

**Comment by MinecraftPlaye** [comment]
Any news on this one?

**Comment by G-Rath** [comment]
@MinecraftPlaye I resolved the merge conflict as requested. For some reason Akxe's comment about adding sidedness information didn't seem to make it through my email system, so I've only just seen it.

Unfortunately I've been too busy with other projects to work on anything Minecraft or Forge related, so I don't know which side the event takes place on, nor really have the time right now to look into it.

I'm happy to commit the desired line to the PR, if someone can provide me with it :)

**Comment by MinecraftPlaye** [comment]
I don't really know on which side the method is beeing called but after a quick search with eclipse I **guess** it is called server side (using Forge for Minecraft 1.11.2):

```
net.minecraft.block.BlockFire
net.minecraft.block.BlockRedstoneDiode
net.minecraft.block.BlockStairs
net.minecraft.block.BlockTNT
net.minecraft.client.multiplayer.PlayerControllerMP
net.minecraft.init -> new BehaviorDispenseOptional
net.minecraft.server.management.PlayerInteractionManager
```
Otherwhise I would look [there ](https://mcforge.readthedocs.io/en/latest/concepts/sides/) to see if anything is written there which might be helpful.

**Pull request merged**